metadata-dao-impl/neo4j-dao/src/main/java/com/linkedin/metadata/dao/Neo4jUtil.java

```diff
@@ -44,6 +46,7 @@ private Neo4jUtil() {
     {
       put(CorpuserUrn.ENTITY_TYPE, getType(CorpUserEntity.class));
       put(CorpGroupUrn.ENTITY_TYPE, getType(CorpGroupEntity.class));
+      put(DataProcessUrn.ENTITY_TYPE, getType(DataProcessEntity.class));
```

Comment:
This map was a quick hack :) We're getting rid of this with this PR: https://github.com/linkedin/datahub/pull/1700